test_regress/t/t_instrumentation.out

```diff
@@ -0,0 +1,101 @@
+$time: 0 | Output outa: 0 | Output outb: 50
+$time: 1 | Output outa: 0 | Output outb: 50
+$time: 2 | Output outa: 0 | Output outb: 50
```

Comment:
Check the callbacks are doing what they need to do, and get different outputs at different times too.